CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+# TiDB Operator v1.0.4 Release Notes
+
+## v1.0.4 What's New
+
+### Action Required
+
+There is no action required if you are upgrading from [v1.0.3](#tidb-operator-v103-release-notes).
+
+### Highlights
+
+[#1202](https://github.com/pingcap/tidb-operator/pull/1202) introduced `HostNetwork` support, which offers better performance comparing to the Pod network. Check out our [benchmark report](https://pingcap.com/docs/dev/benchmark/sysbench-in-k8s/#pod-network-vs-host-network) for details.
+
+> **Note:**
+>
+> Due to [this issue of Kubernetes](https://github.com/kubernetes/kubernetes/issues/78420), the Kubernetes cluster must meet one of the following versions to enable `HostNetwork` of TiDB cluster:
+>  - `v1.13.11` or later
+>  - `v1.14.7` or later
+>  - `v1.15.4` or later
+>  - any version since `v1.16.0`
+
+[#1175](https://github.com/pingcap/tidb-operator/pull/1175) add `podSecurityContext` support for TiDB cluster Pods, we recommend setting the namespaced kernel parameters for TiDB cluster Pods according to our [Environment Recommendation](https://pingcap.com/docs/dev/tidb-in-kubernetes/deploy/prerequisites/#the-configuration-of-kernel-parameters).
+
+New Helm chart `tidb-lightning` brings [TiDB Lightning](https://pingcap.com/docs/stable/reference/tools/tidb-lightning/overview/) support for TiDB in Kubernetes. Check out the [document](https://pingcap.com/docs/dev/tidb-in-kubernetes/maintain/lightning/) for detailed user guide.
+
+Another new Helm chart `tidb-drainer` brings multiple drainers support for TiDB Binlog in Kubernetes. Check out the [document](https://pingcap.com/docs/dev/tidb-in-kubernetes/maintain/tidb-binlog/#deploy-multiple-drainers) for detailed user guide.
+
+### Improvements
+
+- HostNetwork support ([#1202](https://github.com/pingcap/tidb-operator/pull/1202))
+- Support configuring sysctls for Pods and enable net.* ([#1175](https://github.com/pingcap/tidb-operator/pull/1175))
+- Add tidb-lightning support ([#1161](https://github.com/pingcap/tidb-operator/pull/1161))
+- Add new helm chart tidb-drainer to support multiple drainers ([#1160](https://github.com/pingcap/tidb-operator/pull/1160))
+
+## Detailed Bug Fixes and Changes
+
+- Add e2e scripts and simplify e2e jenkins file ([#1174](https://github.com/pingcap/tidb-operator/pull/1174))
+- Fix the pump/drainer data dir to avoid data loss caused by bad configuration([#1183](https://github.com/pingcap/tidb-operator/pull/1183))
+- Add e2e to init sql ([#1199](https://github.com/pingcap/tidb-operator/pull/1199))
```

Comment:
```suggestion
- Add init sql case to e2e ([#1199](https://github.com/pingcap/tidb-operator/pull/1199))
```

Reply:
Not yet, I'm still working on stability test in IDC and hostnetwork test in AWS